virttest/qemu_capabilities.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module for defining the capabilities of vm.
+
+Available class:
+- Flags: Enumerate of the flags of VM capabilities.
+- Capabilities: Representation of VM capabilities.
+"""
+
+__id = 0
```

Comment:
Suggest using `itertools.count`

https://docs.python.org/3.7/library/itertools.html?highlight=itertool#itertools.count

Reply:
yes, thanks for your comment.